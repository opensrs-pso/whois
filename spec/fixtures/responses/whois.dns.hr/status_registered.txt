domain:    google.hr
descr:     Google Hrvatska d.o.o.
descr:     Primorska 26
descr:     10 000 Zagreb
expires:   20120921
source:    DNSHR

