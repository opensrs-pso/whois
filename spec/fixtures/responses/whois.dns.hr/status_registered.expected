#disclaimer
  should: %s raise_error(Whois::AttributeNotSupported)


#domain
  should: %s == "google.hr"

#domain_id
  should: %s raise_error(Whois::AttributeNotSupported)


#status
  should: %s == :registered

#available?
  should: %s == false

#registered?
  should: %s == true


#created_on
  should: %s raise_error(Whois::AttributeNotSupported)

#updated_on
  should: %s raise_error(Whois::AttributeNotSupported)

#expires_on
  should: %s CLASS(time)
  should: %s == Time.parse("2012-09-21")


#registrar
  should: %s raise_error(Whois::AttributeNotSupported)

#registrant_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type          == Whois::Record::Contact::TYPE_REGISTRANT
  should: %s[0].id            == nil
  should: %s[0].name          == "Google Hrvatska d.o.o."
  should: %s[0].organization  == nil
  should: %s[0].address       == "Primorska 26"
  should: %s[0].city          == "Zagreb"
  should: %s[0].zip           == "10 000"
  should: %s[0].state         == nil
  should: %s[0].country       == nil
  should: %s[0].country_code  == nil
  should: %s[0].phone         == nil
  should: %s[0].fax           == nil
  should: %s[0].email         == nil

#admin_contacts
  should: %s raise_error(Whois::AttributeNotSupported)

#technical_contacts
  should: %s raise_error(Whois::AttributeNotSupported)


#nameservers
  should: %s raise_error(Whois::AttributeNotSupported)
